network-config-analyzer/NetworkConfig.py

```diff
@@ -324,11 +344,12 @@ def _allowed_xgress_conns(self, from_peer, to_peer, is_ingress):
         """
         allowed_conns = ConnectionSet()
         denied_conns = ConnectionSet()
+        ingress_denied_conns = ConnectionSet()
         pass_conns = ConnectionSet()
 
```

Comment:
```suggestion

        consider_ingress_resource_policies = False        
        if not is_ingress:
            for policy in self.ingress_deny_policies:
                consider_policy = from_peer in policy.selected_peers
                consider_ingress_resource_policies |= consider_policy
                if consider_policy:
                    policy_conns = policy.allowed_connections(from_peer, to_peer, is_ingress)
                    ingress_denied_conns |= policy_conns.denied_conns

```

Reply:
Suggesting to separate the calculation of the Ingress resource denied connections